### PR TITLE
Java 15032

### PR DESCRIPTION
**Pull request by Bipinkumar27**

### CODE REVIEW

File: apache-httpclient4/src/test/java/com/baeldung/httpclient/HttpClientHeaderV4LiveTest.java [hunk 1]
```diff
@@ -1,15 +1,23 @@
 package com.baeldung.httpclient;
 
+import static org.apache.http.HttpHeaders.USER_AGENT;
```
Comment (dkapil) [inline apache-httpclient4/src/test/java/com/baeldung/httpclient/HttpClientHeaderV4LiveTest.java:3]: revert this change, we will not be adding the v4.5 example

**Review by dkapil — CHANGES_REQUESTED** [review]

File: apache-httpclient4/src/test/java/com/baeldung/httpclient/HttpClientHeaderV4LiveTest.java (hunk 1 above)
Comment (Bipinkumar27) [reply apache-httpclient4/src/test/java/com/baeldung/httpclient/HttpClientHeaderV4LiveTest.java:3]: done
